package.json

```diff
@@ -94,24 +94,34 @@
 		"npm": "^7.0.0"
 	},
 	"devDependencies": {
+		"@babel/eslint-parser": "^7.17.0",
 		"@cypress/browserify-preprocessor": "^3.0.2",
 		"@cypress/webpack-preprocessor": "^5.11.1",
 		"@nextcloud/babel-config": "^1.0.0",
 		"@nextcloud/browserslist-config": "^2.2.0",
-		"@nextcloud/eslint-config": "^7.0.2",
+		"@nextcloud/eslint-config": "^8.0.0",
+		"@nextcloud/eslint-plugin": "^2.0.0",
 		"@nextcloud/stylelint-config": "^2.1.2",
 		"@nextcloud/webpack-vue-config": "^5.0.0",
 		"@vue/test-utils": "^1.3.0",
 		"@vue/vue2-jest": "^27.0.0-alpha.4",
 		"babel-core": "^7.0.0-bridge.0",
```

Comment:
Do we still need the package?

![image](https://user-images.githubusercontent.com/1561347/166477132-c8b70072-5ff6-4305-84bb-3c033733fa20.png)